packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -29,8 +29,8 @@ class AppBarTheme with Diagnosticable {
   /// Creates a theme that can be used for [ThemeData.appBarTheme].
   const AppBarTheme({
     this.brightness,
-    this.color,
-    this.backgroundColor,
+    Color? color,
```

Comment:
Do we want to mark this as deprecated?

Reply:
I'm going to update some existing use cases first. After that this (and a few other parameters) will definitely be deprecated.